CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* docs: Add debugging scripts and documentation [#1839](https://github.com/lambdaclass/cairo-vm/pull/1839)
```

Comment:
Documentation and this kind of scripts that don't modify the VM doesn't need a line in the changelog
Remove it

Reply:
Done. I added "documentation" label to ignore the changelog check.